ns-3_adapter/package.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
```

Comment:
Copyright notice, I'd recommend just doing a pass on all relevant files to check for this.